frontend/src/metabase/admin/permissions/components/PermissionsEditor/PermissionsEditor.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState, useMemo } from "react";
+import PropTypes from "prop-types";
+import { t } from "ttag";
+import { Box } from "grid-styled";
+
+import { PermissionsTable } from "../PermissionsTable";
+import Subhead from "metabase/components/type/Subhead";
+import Text from "metabase/components/type/Text";
+import TextInput from "metabase/components/TextInput";
+import Icon from "metabase/components/Icon";
+import EmptyState from "metabase/components/EmptyState";
+
+import { PermissionsEditorRoot } from "./PermissionsEditor.styled";
+import { PermissionsEditorBreadcrumbs } from "./PermissionsEditorBreadcrumbs";
+
+export const permissionEditorPropTypes = {
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string,
+  columns: PropTypes.array,
+  entities: PropTypes.array,
+  filterPlaceholder: PropTypes.string.isRequired,
+  onChange: PropTypes.func,
+  onSelect: PropTypes.func,
+  onAction: PropTypes.func,
+  onBreadcrumbsItemSelect: PropTypes.func,
+  breadcrumbs: PropTypes.array,
+};
+
+export function PermissionsEditor({
+  title,
+  description,
+  entities,
+  columns,
+  filterPlaceholder,
+  breadcrumbs,
+  onBreadcrumbsItemSelect,
+  onChange,
+  onSelect,
+  onAction,
+}) {
+  const [filter, setFilter] = useState("");
+
+  const handleFilterChange = text => setFilter(text);
```

Comment:
this function looks redundant to me. you should be able to directly pass `setFilter`